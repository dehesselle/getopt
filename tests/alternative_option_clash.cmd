#!/bin/sh
"$1" -a -o abcde -l abcde -- -a -ab -abc -ac
