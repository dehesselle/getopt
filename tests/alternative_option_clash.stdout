 -a --abcde --abcde -a -c --
